keyboards/bastardkb/charybdis/readme.md

```diff
@@ -114,6 +143,20 @@ This behavior can be further customized with the following defines:
 #define CHARYBDIS_SNIPING_DPI_CONFIG_STEP 100
 ```
 
+### Acceleration
+
+By default, the pointer's movements are linear. To enable acceleration, add the following define:
+
+```c
+#define CHARYBDIS_POINTER_ACCELERATION_ENABLE
+```
+
+The acceleration factor can be further tune _via_ the `CHARYBDIS_POINTER_ACCELERATION_FACTOR`:
+
+```c
+#define CHARYBDIS_POINTER_ACCELERATION_FACTOR 24
+```
+
```

Comment:
This was removed in https://github.com/Bastardkb/bastardkb-qmk/commit/96c052e161e02b7e2c56fd4e047f69438cc3658c.  Did you mean to add it back?

Reply:
no I must have based off an outdated branch, I'll remove this.